### ISSUE TITLE
Add ability to sort output results

### ISSUE THREAD

**Issue by fbuys** [fastruby/skunk#116]
**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**

**Before we start...:**

- [x] I checked the documentation and didn't find this feature
- [x] I checked to make sure that this feature has not already been requested

**Branch/Commit:**

Inform what branch/commit/version of "Skunk" you are using.

https://github.com/fastruby/skunk/commit/77bbd4e53bb88981956381f0bbb86b11971db046

**Describe the feature:**

Please include a detailed description of the feature you are requesting and any details on it’s expected behavior.

> **As an end user of skunk**
> **I generate a skunk report for an application**
> **And I see a skunk report sorted in ascending order of skunk scores**

**Problem:**

Please include a detailed description of the problem this feature would solve.

> **As an end user of skunk**
> **I want to generate a skunk report in a different order, say in ascending order of churn_times_cost values**
> **So that I can a analyze my application from different perspectives and emphasizing different metrics**

**I will abide by the [code of conduct](https://github.com/fastruby/skunk/blob/main/CODE_OF_CONDUCT.md)**

**Comment by fbuys** [comment]
1. One solution would be to provide an option that sets the sorting order of the skunk report
2. Another option would be to provide an option that sets the output format to something like CSV that can easily be imported into a spreadsheet application and then be sorted.

**Comment by jack-adorbs** [comment]
Hello, I am curious to help out at some point. Just to clarify are you requesting that it is the files with their associated score that you want to be in ascending order or some other column?

**Comment by FiggyFoxFunc** [comment]
I will have a look through and get more familiar with the code base and then try to make some changes.

Thanks.

**Comment by poonambhagaur61-tech** [comment]
Hi! This is my first open-source contribution. Could you please assign this issue to me? It would really help me gain valuable experience. Thank you!

**Comment by JuanVqz** [comment]
> Hi! This is my first open-source contribution. Could you please assign this issue to me? It would really help me gain valuable experience. Thank you!

All yours

**Comment by poonambhagaur61-tech** [comment]
Hi @JuanVqz, the PR is ready for review. The HTML report now supports sorting by all table columns in ascending and descending order.

**Comment by poonambhagaur61-tech** [comment]
Thanks for the feedback. I updated the report to show the initial sort direction. Since the report starts sorted by Skunk Score in descending order, the Skunk Score column now displays the descending arrow initially. The other columns remain neutral until selected. I also kept the numeric and alphabetical sorting behavior as before.

**Comment by JuanVqz** [comment]
Heads up: #142 adds sorting to the HTML report (`skunk_overview.html`) only, so this issue will be closed when it merges. The console and JSON reports keep their current order (Skunk Score, descending). If you'd like sorting for those formats, please open a new issue.
